spec/controllers/email_alert_subscriptions_controller_spec.rb

```diff
@@ -53,77 +53,49 @@
   end
 
   describe 'POST "#create"' do
-    let(:finder) { govuk_content_schema_example('finder').to_hash.merge(title: 'alert-name') }
-
     before do
-      content_store_has_item('/cma-cases', finder)
-      content_store_has_item('/cma-cases/email-signup', signup_finder)
+      content_store_has_item('/cma-cases', cma_cases_content_item)
+      content_store_has_item('/cma-cases/email-signup', cma_cases_signup_content_item)
     end
 
     context "finder has default filters" do
-      let(:signup_finder) {
-        cma_cases_signup_content_item.to_hash.merge("details" => {
-          "filter" => { "content_purpose_supergroup" => 'news-and-communications' },
-        })
-      }
+      it "fails if the relevant filters are not provided" do
```

Comment:
The test says `it fails`, but the expectation is for a successful response.  Which way should it be?

Reply:
I think the point here is that the 'new' template is being rendered which indicates a failure, i.e. the previous page is being re-rendered.